circle.yml

```diff
@@ -18,18 +21,28 @@ dependencies:
     # node_modules is included by default
 
   override:
+    - pip install tox mozdownload mozinstall
     - google-chrome --version
     - ./bin/circleci/install-node-dependencies.sh
-    - ./bin/circleci/setup-test-dependencies.sh
     - ./bin/circleci/build-addon.sh
 
 test:
+  pre:
+    - sudo apt-get upgrade firefox
+    - wget -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.16.0/geckodriver-v0.16.0-linux64.tar.gz
+    - gunzip -c geckodriver.tar.gz | tar xopf -
+    - chmod +x geckodriver
+    - sudo mv geckodriver /home/ubuntu/bin
+    - firefox --version
+    - geckodriver --version
   override:
     - ./bin/circleci/test-addon.sh
     - ./bin/circleci/test-frontend.sh
-    - ./bin/circleci/run-integration-tests.sh
+    - tox
   post:
     - bash <(curl -s https://codecov.io/bash)
+    - cp geckodriver.log $CIRCLE_ARTIFACTS
```

Comment:
With the latest version of pytest-selenium this will be `driver.log`, however the upcoming release will change this by creating a log for each driver instance and storing them in tmp. It'll probably be safest to use the log integrated with the HTML report.